lib/verify/response.rb

```diff
@@ -69,7 +69,7 @@ def full_name
     end
 
     def most_recent_verified_value(attributes)
-      return if attributes.empty?
+      return if attributes.blank?
```

Comment:
Sounds like what you're saying is that this guard is only meant to trigger when `attributes` is `nil`. In which case, would it make more sense for this to read `attributes.nil?`?

Reply:
It's possible for `middleNames` to come back as an empty array though